### DISCUSSION TITLE
How do I turn these blue texts showing the argument : parameter info?

### DISCUSSION THREAD

**Discussion by 17Swagat** [neoclide/coc.nvim#4055] [Q&A] [answered]
How do I turn these blue texts showing the argument : parameter info...
These things are making my code look messy.

![coc_prob](https://user-images.githubusercontent.com/72545910/184498084-3eb1df63-dc89-440d-8917-d84c51e0c44c.png)

**Comment by joshua7v** [comment, marked as answer]
if you are using coc-clangd, you could turn off hints by

```
"clangd.inlayHints.enable": false
```

**Reply by doorz1009** [reply]
Coc does not seem to respect this setting in my Coc Config. The only way I can get them to go way is by using `:CocCommand clangd.inlayHints.toggle`
